doc/api/deprecations.md

```diff
@@ -2555,6 +2570,7 @@ Passing a callback to [`worker.terminate()`][] is deprecated. Use the returned
 [`process.env`]: process.html#process_process_env
 [`punycode`]: punycode.html
 [`require.extensions`]: modules.html#modules_require_extensions
+[`ReadStream.open()`]: fs.html#fs_class_fs_readstream
```

Comment:
This needs to be placed after the `` [`REPLServer.clearBufferedCommand()`] `` (ASCII sorting is case-sensitive).